### PR TITLE
feat(sui-bundler): translate to english

### PR DESCRIPTION
**Pull request by miduga**
Translate the README to english.

Fixes https://github.com/SUI-Components/sui/issues/211

### CODE REVIEW

File: packages/sui-bundler/README.md [hunk 1]
```diff
@@ -108,9 +101,9 @@ register({
 })()
 ```
 
-Debes propocionar un handler para cuando se cachea por primera vez contenido y otro para cuando hay nuevo contenido cacheado y tienes que refrescar la página para poder disfrutarlo.
+You should pass a handler in order to handle when it gets cached for the first time the content and another when you get new content and want to handle how to show a notification to the user in order to let him decide if he wants to refresh the page.
```
Comment (davidbarna) [inline packages/sui-bundler/README.md]: You should pass a handler in order to handle when content gets cached for the first time

File: packages/sui-bundler/README.md [hunk 2]
```diff
@@ -153,19 +146,14 @@ Puedes configurar que quieres cachear para ser usado offline:
 }
 ```
 
-runtime usa la mismo API que (sw-toolbox)[https://googlechrome.github.io/sw-toolbox/]
-
-además whitelist es un listado de regexp que indican que páginas son seguras para hacer solo CSR. Puedes usar el string `::all::` para indicar que siempre quieres usar CSR.
-
-Si no defines un listado whitelist. Siempre se hará SSR
+Runtime follows the (API of sw-toolbox)[https://googlechrome.github.io/sw-toolbox/]. Also, the whitelist is a list of regexp that indicates which pages are secure to use only Client Server Rendering. You could use the `::all::` string for indicating that you always want to use Client Side Rendering.
```
Comment (davidbarna) [inline packages/sui-bundler/README.md]: FYI dead link: use https://github.com/GoogleChromeLabs/sw-toolbox

"to use only Client" => "to use only with Client" ?

string for indicating that you always want to use Client Side Rendering. => string  to indicate that you always want to use...

File: packages/sui-bundler/README.md [hunk 3]
```diff
@@ -153,19 +146,14 @@ Puedes configurar que quieres cachear para ser usado offline:
...
 
 ## Externals
 
-Cumple la función de subir una librería external a tu proyecto que normalmente lo pondrías a mano con su propio tag script, a tu cdn, y colocar una referencia a el en index.html y con un hash que representa el contenido del fichero.
+It offers you a way to upload an external library to your project that you normally will put by hand in a tag script in the index.html file. It adds a reference in the index.html with a hash.
```
Comment (davidbarna) [inline packages/sui-bundler/README.md]: that you would normally put

File: packages/sui-bundler/README.md [hunk 4]
```diff
@@ -153,19 +146,14 @@ Puedes configurar que quieres cachear para ser usado offline:
...
 
 ## Hot Module Replacement - React
-Lo tienes todo preparado para poder usarlo en tus proyectos de react.
-Solo tienes que aplicar estas instrucciones en tu `app.js` https://webpack.js.org/guides/hmr-react/#code
+It offers Hot Module Replacement out-of-the-box, you only has to follow these instructions for your project: https://webpack.js.org/guides/hmr-react/#code
```
Comment (davidbarna) [inline packages/sui-bundler/README.md]: you only have to follow

**Comment by miduga** [comment]
:+1: done @davidbarna

**Pull request merged**